src/components/NavBar/NavBar.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import classNames from 'classnames';
+
+export const NavBar: React.FC = () => (
+  <nav
+    data-cy="nav"
+    className="navbar is-fixed-top has-shadow"
+    role="navigation"
+    aria-label="main navigation"
+  >
+    <div className="container">
+      <div className="navbar-brand">
+        <NavLink
+          className={({ isActive }) => classNames(
+            'navbar-item ',
+            {
+              'has-background-grey-lighter': isActive,
+            },
+          )}
+          to="/"
+        >
+          Home
+        </NavLink>
+
+        <NavLink
+          className={({ isActive }) => classNames(
+            'navbar-item ',
+            {
+              'has-background-grey-lighter': isActive,
+            },
+          )}
+          to="/people"
+        >
+          People
+        </NavLink>
```

Comment:
I would to create separate component for `NavLink` for avoid dublicate code) but up for you)

Reply:
agree, let's create a separate component for `NavLink`